src/clusterfuzz/_internal/bot/tasks/utasks/uworker_io.py

```diff
@@ -428,18 +404,24 @@ def save_rich_type(self, attribute, value):
     field.CopyFrom(wrapped_entity_proto)
 
 
-class BuildData(UworkerMsg):
-  """Class representing an unserialized ProgressionTaskOutput message from
+class FuzzTaskOutput(UworkerOutput):
```

Comment:
It's weird to me that fields of uworker output would subclass `UworkerOutput`. The inheritance "is a" relationship does not hold: a `BuildData` object *is not a* `UworkerOutput` object as well. `UworkerMsg` seems to provide the base functionality we would like to reuse here? If not, could we extend it to provide the right functionality?

Reply:
+1